client/layout/masterbar/logged-in.jsx

```diff
@@ -96,6 +96,7 @@ export default React.createClass( {
 					{ this.translate( 'New Post' ) }
 				</Publish>
 				<Item
+					tipTarget="me"
```

Comment:
We were thinking about this, and `pointer`/`target`/`tag` came up, but they felt too generic and HTMLy. What d'you think? ;)